bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/handler/DoorbellHandler.java

```diff
@@ -88,13 +94,19 @@ public class DoorbellHandler extends BaseThingHandler {
 
     private DoorbirdAPI api = new DoorbirdAPI();
 
+    private BundleContext bundleContext;
+
+    private @NonNullByDefault({}) ServiceRegistration<AudioSink> audioSinkRegistration;
```

Comment:
Could be `@Nullable` as you are checking if null when using it.

Reply:
Done